src/embind/embind_shared.js

```diff
@@ -0,0 +1,149 @@
+// Copyright 2023 The Emscripten Authors.  All rights reserved.
+// Emscripten is available under two separate licenses, the MIT license and the
+// University of Illinois/NCSA Open Source License.  Both these licenses can be
+// found in the LICENSE file.
+  $InternalError__postset: "InternalError = Module['InternalError'] = class InternalError extends Error { constructor(message) { super(message); this.name = 'InternalError'; }}",
```

Comment:
Can we make this into a more "normal" library file with it own `mergeInto`.

I think it can still be #included, but at the top level, and not within a JS object.